sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/MiscellaneousOperationsLiveTests.cs

```diff
@@ -48,13 +48,13 @@ public async Task GetResourceDetails()
         [RecordedTest]
         [TestCase(true)]
         [TestCase(false)]
+        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/38635")]
```

Comment:
The issue in this test was not introduced with the changes in this PR, they only exposed a test bug that was not being triggered before. Issue https://github.com/Azure/azure-sdk-for-net/issues/38635 tracks it.